gwosc/datasets.py

```diff
@@ -230,6 +230,7 @@ def _matched(iter_):
             segment=segment,
             host=host,
             version=version,
+            catalog=catalog
```

Comment:
Can you please add a trailing comma, so that this line stands on its own, and adding another keyword argument in the future does not require editing this line.

```suggestion
            catalog=catalog,
```